cloudflare/ai/call.go

```diff
@@ -0,0 +1,35 @@
+//go:build !js
+
+package ai
+
+import (
+	"fmt"
+	"io"
+)
+
+type AI struct {
+	instance string
+}
+
+func New(varName string) (*AI, error) {
+	fmt.Println("NewNamespace called with varName:", varName)
```

Comment:
using fmt.Println makes this a verbose logging that cannot be turned off - maybe consider using `slog` since it is in a standard library? :)

Reply:
i put for my debug... i remove as in other part of the library...